templates/react/api/_entrypoint.js

```diff
@@ -0,0 +1,2 @@
+export const API_HOST = 'http://localhost';
```

Comment:
`http://localhost` shouldn't been hardcoded. The generated file should contain the host and the path matching the one used to retrieve Hydra docs.